packages/psss/src/api/queries/useArchivedAlerts.js

```diff
@@ -0,0 +1,26 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+
+import { MIN_DATE, SYNDROMES } from '../../constants';
+import { getPeriodByDate } from '../../utils';
+import { useReport } from './helpers';
+
+export const useArchivedAlerts = orgUnitCodes => {
```

Comment:
again it seems like this could be drier. I don't believe in 100% dry all the time but this is a good candidate. I guess the main reason not combine them would be if we anticipate different requirements in the future between the archive data and the active data which may well be the case.

Reply:
Yeap, that was my line of thought too. There already exist some subtle differences (start end end dates), and the queries are simple enough that re-defining them probably makes them simpler than abstracting them.

Reply:
You know what, you are right - and actually that would reflect the original endpoint design:

Active alerts: `/alerts`
Archived alerts: `/alerts/archived`

Thanks for pointing this out - will change that!